modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -304,3 +304,71 @@
 	attack_verb_continuous = list("caws","skrees","pecks")
 	attack_verb_simple = list("caw","skree","peck")
 	squeak_override = list('modular_skyrat/modules/emotes/sound/voice/peep_once.ogg' = 1,'modular_skyrat/modules/emotes/sound/voice/caw.ogg' = 1,'modular_skyrat/modules/emotes/sound/voice/bawk.ogg' = 1,'modular_skyrat/modules/emotes/sound/emotes/voxscream.ogg' = 1)
+
+/obj/item/toy/plush/zapp
+	name = "Lil' Zapp"
+	desc = "An authentic piece of primo Pwr Game merchandise! \
+			This cuddly companion is the perfect ornament to decorate your battlestation. \
+			He sits upright unassisted, and can hold your headset, webcam, or keep your Pwr Game safe and secure. \
+			This one is outfitted with a state-of-the-art skill reader; \
+			just squeeze him tight and Zapp will tell you if you're ready for the next big game!"
+	icon = 'modular_skyrat/master_files/icons/obj/plushes.dmi'
+	icon_state = "plushie_zapp"
+	inhand_icon_state = "plushie_zapp"
+	attack_verb_continuous = list("boops","nuzzles")
+	attack_verb_simple = list("boop", "nuzzle")
+	squeak_override = list('sound/effects/can_open1.ogg' = 1, 'sound/effects/can_open2.ogg' = 1, 'sound/effects/can_open3.ogg' = 1)
+	var/static/list/skill_response = list(
+		"Weak! What are you, a mobile gamer?",
+		"Come on, you can do better than that! Play some Orion Trial and try again.",
+		"Hey, not bad! Try and work on your APM.",
+		"Nice! You should see about competing in some local tournaments, gamer!",
+		"Now that's real skill! I think you deserve some Pwr Game.",
+		"Gamer God in the house! Look upon them and weep, console peasants!",
+		"Whoa! Gamer overload! Stand clear!!",
+	)
+	var/static/list/hit_response = list(
```

Comment:
I know that this is a plushie, but please document those new vars anyway :)

Reply:
sure.